data_management/prov.py

```diff
@@ -643,28 +647,28 @@ def generate_prov_document(data_product, depth, request):
 
     """
     url = request.build_absolute_uri('/')
-    cenral_registry_url = settings.CENTRAL_REGISTRY_URL
-    if not cenral_registry_url.endswith('/'):
-        cenral_registry_url = f'{cenral_registry_url}/'
+    central_registry_url = settings.CENTRAL_REGISTRY_URL
+    if not central_registry_url.endswith('/'):
+        central_registry_url = f'{central_registry_url}/'
 
     doc = prov.model.ProvDocument()
 
-    if url == cenral_registry_url:
+    if url == central_registry_url:
         # we are using the main registry
         reg_uri_prefix = 'reg'
-        doc.add_namespace(reg_uri_prefix, cenral_registry_url)
+        doc.add_namespace(reg_uri_prefix, central_registry_url)
     else:
         # we are using a local registry
         reg_uri_prefix = 'lreg'
         doc.add_namespace(reg_uri_prefix, url)
 
     # the vocab namespace is always the main registry
-    doc.add_namespace(FAIR_VOCAB_PREFIX, f'{cenral_registry_url}vocab/#')
+    doc.add_namespace(FAIR_VOCAB_PREFIX, f'{central_registry_url}vocab/#')
     # we need to tell SONAR to ignore 'http' in the vocab URLs
-    doc.add_namespace(DCAT_VOCAB_PREFIX, 'http://www.w3.org/ns/dcat#')  # NOSONAR
-    doc.add_namespace(DCMITYPE_VOCAB_PREFIX, 'http://purl.org/dc/dcmitype/')  # NOSONAR
-    doc.add_namespace(DCTERMS_VOCAB_PREFIX, 'http://purl.org/dc/terms/')  # NOSONAR
-    doc.add_namespace(FOAF_VOCAB_PREFIX, 'http://xmlns.com/foaf/spec/#')  # NOSONAR
+    doc.add_namespace(DCAT_VOCAB_PREFIX, DCAT_VOCAB_NAMESPACE)  # NOSONAR
```

Comment:
remove the `# NOSONAR` from these lines and add them to end of the *_NAMESPACE lines at the top of the file

Reply:
ah, great, will do